modules/claims_api/lib/pdf_generator_service/pdf_client.rb

```diff
@@ -12,12 +17,28 @@ def initialize(request_body = nil)
     end
 
     def generate_pdf
+      resp = client.post('526', @request_body).body
+      log_outcome_for_claims_api('pdf_generator', 'success', resp.is_a?(String))
+
+      resp
+    rescue => e
+      detail = e.respond_to?(:original_body) ? e.original_body : e
+      log_outcome_for_claims_api('pdf_generator', 'error', detail)
+
+      raise ::ClaimsApi::Common::Exceptions::Lighthouse::BadRequest.new(
+        detail: JSON.parse(e.original_body).deep_symbolize_keys
```

Comment:
`detail` creates a weird double nesting here. instead of our errors looking like the LH standards of 
```
{
    "errors": [
        {
            "title": "Unsupported Media Type",
            "status": "400",
            "detail": "Required request body is missing: public byte[] gov.va.api.lighthouse.benefits.form526.service.pdfgeneration.PdfGenerationV1Controller.generate(gov.va.api.lighthouse.benefits.form526.api.v1.PdfRequestV1)"
        }
    ]
}
```

I'm seeing more of a "double" effect
```
{
    "errors": [
        {
            "title": "Bad Request",
            "status": "400",
            "detail": {
                "errors": [
                    {
                        "id": "r34l-1d",
                        "status": "415",
                        "code": "HttpMessageNotReadable",
                        "title": "blah blah blah",
                        "detail": "blah blah blah",
                        "meta": {
                            "time": 1716920721702,
                            "diagnostics": "blah blah blah"
                        }
                    }
                ]
            },
            "source": {
                "pointer": "/modules/claims_api/lib/pdf_generator_service/pdf_client.rb:29:in `rescue in generate_pdf'"
            }
        }
    ]
}
```